src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public enum Colors {
+    Blue,
+    Red,
+    Green,
+    Yellow,
+    Brown,
+    White,
+    Black

```

Comment:
Enum constants should be upper case and snake case styled.

Reply:
not fixed